edge/evtstreams/cpu2evtstreams/PolicyRegister.md

```diff
@@ -130,48 +87,64 @@ hzn exchange service listpolicy <published-cpu2evtstreams-service-name>
 
 - Business Policy, like the other two Policy types, contains a set of `properties` and a set of `constraints`, but it contains other things as well. For example, it explicitly identifies the Service it will cause to be deployed onto Edge Nodes if negotiation is successful, in addition to configuration variable values, performing the equivalent function to the `-f horizon/userinput.json` clause of a Deployment Pattern `hzn register ...` command. The Business Policy approach for configuration values is more powerful because this operation can be performed centrally (no need to connect directly to the Edge Node).
 
-- Below is the `horizon/business_policy.json` file used for this example:
+- Below is the `business_policy.json` file you grabbed in step one:
 
 ```bash
 {
-  "label": "$SERVICE_NAME Business Policy for $ARCH",
+  "label": "ibm.cpu2evtstreams Business Policy for $ARCH",
```

Comment:
Instead of removing the variables, you could also have them wget the `hzn.json` file and eval it like `eval $(hzn util configconv -f horizon/hzn.json)`

Reply:
done